ICSharpCode.CodeConverter/VB/NodesVisitor.cs

```diff
@@ -1255,7 +1258,7 @@ public override VisualBasicSyntaxNode VisitCastExpression(CSS.CastExpressionSynt
                     return SyntaxFactory.PredefinedCastExpression(SyntaxFactory.Token(SyntaxKind.CBoolKeyword), expr);
                 case SpecialType.System_Char:
                     return sourceType?.IsNumericType() == true
-                        ? (VisualBasicSyntaxNode)SyntaxFactory.InvocationExpression(SyntaxFactory.ParseName(nameof(Strings.ChrW)), ExpressionSyntaxExtensions.CreateArgList(expr))
+                        ? (VisualBasicSyntaxNode)SyntaxFactory.InvocationExpression(SyntaxFactory.ParseName(GetQualifiedName(nameof(Microsoft), nameof(Microsoft.VisualBasic), nameof(Microsoft.VisualBasic.Strings.ChrW))), ExpressionSyntaxExtensions.CreateArgList(expr))
```

Comment:
In general, it's best to full qualify, i.e. add nameof(Microsoft.VisualBasic.Strings) in there too. Though the chances of someone declaring another Microsoft.VisualBasic namespace with a direct ChrW member seem very low I'll admit.